jerry-core/debugger/jerry-debugger-ws.c

```diff
@@ -13,8 +13,6 @@
  * limitations under the License.
  */
 
-#ifdef JERRY_DEBUGGER
```

Comment:
It cannot. You put JERRY_DEBUGGER after #include <arpa/inet.h>, and if that system has no arpa/inet.h (which is not needed if you are not using the debugger), you will get a compile error.